pavelib/paver_tests/test_servers.py

```diff
@@ -251,6 +255,7 @@ def verify_server_task(self, task_name, options, contracts_default=False):
                 node_env="production",
                 static_root_lms=None,
                 static_root_cms=None,
+                js_env_extra_config=json.dumps({}),
```

Comment:
According to the [comment raised earlier](https://github.com/edx/edx-platform/pull/25695/files#r553657732), update the `json.dumps({})` to `json.dumps("{}")` for consistency 👍🏼

Reply:
Addressed 👍🏼